src/ai/threads/messages/message.ts

```diff
@@ -114,7 +118,151 @@ export default class Messages {
     return { ...txResult, messages };
   }
 
-  private async getMessageCreateTxBody(
+  async update(
```

Comment:
(question) does the openai api allow message updates(just for double checking)?

Reply:
the openai api only allows update message `metadata` field, not content field!

Reply:
oh I see.. thanks!